examples/logging/all_logs.md

```diff
@@ -0,0 +1,51 @@
+This example captures all logs sent to the python logger.  This is helpful if you don't know the logger name
+for the loggers config or if you simply want to make sure you collect ALL logs.  But be warned, ALL logs might
+be a lot of logs:
+
+```python
+LOG_LEVEL = "DEBUG" if DEBUG else "INFO"
+LOGGING = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "formatters": {
+        "normal": {
+            "format": "%(asctime)s.%(msecs)03d %(levelname)-7s %(name)s:  %(message)s",
+            "datefmt": "%H:%M:%S",
+        },
+    },
+    "handlers": {
+        "normal_console": {
+            "level": "DEBUG",
+            "class": "logging.StreamHandler",
+            "formatter": "normal",
+        },
+    },
+    "loggers": {
+        "django": {"handlers": ["normal_console"], "level": LOG_LEVEL},
+        "nautobot": {
+            "handlers": ["normal_console"],
+            "level": LOG_LEVEL,
+        },
```

Comment:
Nit: it would be maybe more readable in all of these examples if the loggers were consistently formatted, i.e. either:

```python
        "django": {
            "handlers": ["normal_console"],
            "level": LOG_LEVEL,
        },
        "nautobot": {
            "handlers": ["normal_console"],
            "level": LOG_LEVEL,
        },
```

or

```python
        "django": {"handlers": ["normal_console"], "level": LOG_LEVEL},
        "nautobot": {"handlers": ["normal_console"], "level": LOG_LEVEL},
```